packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts

```diff
@@ -127,15 +130,15 @@ export class ChainAdapter
           }
         ],
         signatures: [],
-        memo: ''
+        memo: memo || ''
```

Comment:
prefer `??` for `undefined` or `null` check
```suggestion
        memo: memo ?? ''
```